requirements.txt

```diff
@@ -1,3 +1,4 @@
 fastapi==0.61.1
 jwt==1.0.0
-uvicorn~=0.12
\ No newline at end of file
+uvicorn~=0.12
+python-multipart==0.0.5
```

Comment:
conflict #23. 

You don't have responsibility for this file. 
So, I recommend to exclude this file from this commit.